mycqu/utils/datetimes.py

```diff
@@ -28,12 +38,13 @@ def parse_weeks_str(string: str) -> List[Tuple[int, int]]:
     return [parse_period_str(unit) for unit in string.split(',')]
 
 
-def parse_weekday_str(string: str) -> int:
-    return WEEKDAY[string]
+def parse_weekday_str(string: str) -> Optional[int]:
+    return SHORT_WEEKDAY.get(string) if SHORT_WEEKDAY.get(string) else LONG_WEEKDAY.get(string)
```

Comment:
```suggestion
    return SHORT_WEEKDAY.get(string) if SHORT_WEEKDAY.get(string) is not None else LONG_WEEKDAY.get(string)
```
或者
```suggestion
    return SHORT_WEEKDAY.get(string, LONG_WEEKDAY.get(string))
```
否则 `parse_weekday_str("一")` 值为 `None`